providers/default.rb

```diff
@@ -191,7 +191,7 @@ def execute_install_script(install_script)
       code <<-EOH
     #{install_script}
       EOH
```

Comment:
should probably action :nothing this here if you're going to force compile time.  not sure it matters, but its best to follow the pattern.